README.md

```diff
@@ -40,7 +40,7 @@ See the section about [deployment](https://facebook.github.io/create-react-app/d
 
 ## Demo
 
-- Open [live demo](https://amal-covid-tracker.herokuapp.com/) 
+- Open [live demo](https://nchorage-03766.herokuapp.com/) 
 
 ## Vidoe
 
```

Comment:
- [ ] Also be sure to add a video that you will describe what you have implemented in your project.


![iii](https://user-images.githubusercontent.com/61562626/142909099-0cfa95a2-fefb-43f9-bd02-f8d96faae918.jpeg)

Reply:
This issue is well resolved! ✔️